.env.template

```
@@ -2,5 +2,6 @@ NEMLI_DISCORD_TOKEN=
 NEMLI_DISCORD_MAX_MESSAGES=
 NEMLI_OPENAI_API_KEY=
 NEMLI_OPENAI_MODEL=
+NEMLI_OPENAI_TEMPERATURE=
```

```suggestion
NEMLI_OPENAI_TEMPERATURE=
NEMLI_REMOVE_STOPWORDS=
```

Esqueci de colocar essa no template em outro PR